projects/hslayers/src/components/add-data/url/add-data-url.service.ts

```diff
@@ -120,7 +120,7 @@ export class HsAddDataUrlService {
     }, 1000);
   }
 
-  searchForChecked(services: Array<any>): boolean {
-    return services.some((service) => service.checked);
+  searchForChecked(services: Array<any>): void {
+    this.hasAllChecked = services.some((service) => service.checked);
```

Comment:
Maybe hasAnyChecked would be better since it only takes one to evaluate to try if we use `some` function